src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -147,6 +155,7 @@ const SendFunds = ({
     const amountValidation = composeValidators(
       required,
       mustBeFloat,
+      minMaxLength(1, 24),
```

Comment:
Just one question, where is the number 24 coming from?
I've briefly googled and it seems ERC-20 allows 18 decimals (so we have 6 extra non-decimal digits?). Maybe we should put it in some config?

Reply:
I understand that "decimals" refers to wei (so basically digits in a number, not real decimals) but this validation happens on the input string. It counts the symbols in the input.

So I guess we should allow entering more symbols in the integer part and limit only the decimal part?

<img width="474" alt="Screenshot 2021-04-20 at 16 52 49" src="https://user-images.githubusercontent.com/381895/115417516-f89e0900-a1f8-11eb-8466-1e22494406d9.png">

Reply:
aren't we going to have the same problem for tokens with less than 24 decimals? or 18? 
Some ETH20 tokens have 6 decimals. USDC if I remember correctly.

we could try to get the exact decimals by using the ERC20 Abi and fetching the decimals.

Reply:
if fetching the decimals throws an error, we could fallback to 24.

Reply:
True, I suppose it won't fully fix #1662.

Reply:
I understand. I think it should be fixed in another PR though (when #1662 is addressed)